.yarnrc.yml

```diff
@@ -5,6 +5,8 @@ enableGlobalCache: true
 logFilters:
   - code: YN0013
```

Comment:
I think this can be removed now. I added it at some point during the migration to reduce the log noise in CI, but now that we cache dependencies between builds this warning will tell us if things are not working as expected.

Reply:
By default Yarn enables `preferAggregateCacheInfo` when running on a CI so all you should see there is something like this:
```
➤ YN0013: │ 3747 packages were already cached
➤ YN0013: │ No packages were cached - 3747 packages had to be fetched
➤ YN0013: │ 3746 packages were already cached, one had to be fetched (@automattic/color-studio@npm:2.5.0)
```
You can enable it manually to have the same effect outside of the CI without discarding the log entry
https://yarnpkg.com/configuration/yarnrc#preferAggregateCacheInfo